packages/luals-annotations/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
+	preset: "ts-jest",
+	testEnvironment: "node",
+	testMatch: ["**/*.test.ts"],
+	collectCoverage: true,
+	collectCoverageFrom: ["src/**/{!(index),}.ts"],
+	coverageThreshold: {
+		global: {
+			lines: 85,
+			branches: 75,
+			functions: 90,
+			statements: 85,
```

Comment:
I DONT KNOW EITHER!!!!!!
Jest says that the luaLSType file is like at 98% branches. I checked the line number it was complaining for and it's just an empty line.
I also set it low because the annotation builder tests weren't done yet. Should be fine shooting for 95-100 now.

Reply:
Done